install/upgrade-postgres.sh

```diff
@@ -0,0 +1,44 @@
+echo "${_group}Ensuring proper PostgreSQL version ..."
+
+if [[ -n "$(docker volume ls -q --filter name=sentry-postgres)" && "$(docker run --rm -v sentry-postgres:/db busybox cat /db/PG_VERSION 2>/dev/null)" == "9.6" ]]; then
+  docker volume rm sentry-postgres-new || true
```

Comment:
I don't like introducing an additional prompt asking for the user to continue with a warning since for people who have automated upgrade scripts it will cause those to start hanging (us and our daily upgrades 😅). It feels like adding an additional install option like we do for `minimize-downtime` and `skip-commit-check` also isn't super applicable here since this is a one time thing too.